test/jdk/ProblemList.txt

```diff
@@ -852,7 +852,6 @@ java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.html 8080185 macosx-a
 javax/swing/JTabbedPane/4666224/bug4666224.html 8144124  macosx-all
 java/awt/event/MouseEvent/AltGraphModifierTest/AltGraphModifierTest.java 8162380 generic-all
 java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java 8163086 macosx-all
-java/awt/image/multiresolution/MultiResolutionIcon/IconTest.java 8250804 macosx-all,linux-all
```

Comment:
Does this test work fine on the macOS? Looks like it was reported here:

https://bugs.openjdk.java.net/browse/JDK-8250804?focusedCommentId=14379402&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14379402

Reply:
O, I see that you made it Linux only.

Reply:
Yes, it does not work on macOS.
For changing dock icon on macOS you should use `java.awt.Taskbar#setIconImage` or assign it with application bundle.